### DISCUSSION TITLE
How to give "Admin Service Accounts" permissions to create portfolios

### DISCUSSION THREAD

**Discussion by robertmarsal** [featurehub-io/featurehub#1049] [Q&A] [answered]
After creating an "Admin Service Account" I can't create a portfolio and receive a "403 Forbidden" error.  I can't find a way to give access to this functionality to an admin service account although the documentation suggests this should be possible: https://docs.featurehub.io/featurehub/latest/admin-development-kit.html#_create_portfolio

Any guidance on how to achieve this? I would like to create an Admin API key that gives me access to all features to be able to manage the application completely via the API including creating portfolios, apps, features, etc. 

Example HTTP call:
```http
POST http://localhost:8085/mr-api/portfolio
Authorization: Bearer XXX
Content-Type: application/json

{
  "name": "My Portfolio"
}
```
Calling the same endpoint with the token I get when I log in as a user does work.

Thank you.

**Comment by IrinaSouth** [comment]
Hey @robertmarsal , thanks for posting this question. 

I have reviewed our logic and you are right, it appears that we intentionally don't allow Admin Service Accounts to have full access such as "Super admin" role, which is the role required to create Portfolios. We think we could tweak this functionality and allow to add this role for Admin Service Accounts. I'll create an issue based on this so we can track in our main repo.

**Comment by rvowles** [comment, marked as answer]
If you need to work around this for the time being, you can login as a superuser, find your API token and just update the Admin SDK user via the API to have super user permissions. It is a UI related limitation.
